### ISSUE TITLE
bug: trace latency is 0 for traces without observations

### ISSUE THREAD

**Issue by marcklingen** [langfuse/langfuse#4773]
### Describe the bug

Currently, traces have a `timestamp` only and their latency is derived from the min/max start/end times of the observations within a trace. This is a good approximation which assumes that traces mainly group observations.

However, this does not represent latency correctly in some usage patterns of the Langfuse integrations. This especially happens in black-box experiments on a dataset without a full tracing setup where a single trace is created for each item but no child observations are logged for each of these.

### To reproduce

Example in which there will be a single trace, name "test_delay", latency 0
```
import time
from langfuse.decorators import observe

@observe()
def test_delay():
   time.sleep(1)
```

### SDK and container versions

Applies to all integrations and Langfuse versions

### Are you interested to contribute a fix for this bug?

No

**Comment by marcklingen** [comment]
next to actually resolving this issue and allowing for native timings on the traces itself, @hassiebp is there an integration-level workaround for this to automatically log a child "root" span with the correct timings?

**Comment by hugh-klein** [comment]
@marcklingen any update here? is there "an integration-level workaround for this to automatically log a child "root" span with the correct timings?"

**Comment by njcolvin** [comment]
Bump. Would be nice to track latency on outermost observed functions. Thanks in advance!

**Comment by marcklingen** [comment]
agree, thanks for the ping. in the meantime, can you add a wrapper span (another observe()) in order to track this?

**Comment by njcolvin** [comment]
Yes this works, thanks for the workaround.

**Comment by marcklingen** [comment]
great, thanks for confirming @njcolvin

**Comment by marcklingen** [comment]
this will be fixed when migrating to the latest langfuse sdks that use the otel endpoint
